project/Dependencies.scala

```diff
@@ -235,4 +243,28 @@ object Dependencies {
     http4sCirce,
     scalaTest
   )
+
+  val wsmDependencies: List[ModuleID] = List(
+    "bio.terra" % "workspace-manager-client" % "0.254.452-SNAPSHOT"
+      exclude("com.sun.activation", "jakarta.activation"),
+    /*
+    1. WSM is looking for the rs-api under javax.*.
```

Comment:
Should this comment be deleted or an actual TODO?

Reply:
oh yup that was taken directly from rawls, but that may have been before I did a fetch update, so I'll check on that and (probably) delete this